lib/brexit_checker/actions.yaml

```diff
@@ -1471,3 +1471,19 @@ actions:
   criteria:
   - chemical
   audience: business
+- id: S033
```

Comment:
This looks like an addition?
Wouldn't there be a notication in this case too?

Reply:
Oops - you're right! That's what happens when I work on three cards at the same time...